core/modules/views/handlers/views_handler_sort_random.inc

```diff
@@ -19,4 +19,8 @@ class views_handler_sort_random extends views_handler_sort {
     parent::options_form($form, $form_state);
     $form['order']['#access'] = FALSE;
   }
+
+  function can_expose() { return FALSE; }
+
+  function admin_summary() { }
```

Comment:
...wondering if we should return something here. Perhaps `return '';`

Reply:
Ah yes, probably. Though I think `return;` will suffice.